src/ReinforcementLearningDatasets/src/atari/atari_dataset.jl

```diff
@@ -0,0 +1,169 @@
+"""
+Represents an iterable dataset of type AtariDataSet with the following fields:
+
+`dataset`: Dict{Symbol, Any}, representation of the dataset as a Dictionary with style as `style`
+`epochs`: Vector{Int}, list of epochs to load
+`repo`: String, the repository from which the dataset is taken
+`size`: Integer, the size of the dataset
```

Comment:
the meaning of size is unclear to me. change another name?

Reply:
size denotes the size of the dataset, It seemed clear to me. Would be great if you could suggest something better.

Something like dataset_size?

Reply:
How about `length` based on the following line:

https://github.com/JuliaReinforcementLearning/ReinforcementLearning.jl/pull/429/files/ca62405e2247e7c5ab9ac0f0b1caa0f16d94255f#diff-ee59e3b8adc8856b61bbc10809639b624390891bbd4366bc262405dc63a1964fR161

Reply:
yep, makes sense